Receiver/Data.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
+#include<iostream>
+#include<string>
+#include<vector>
+using namespace std;
```

Comment:
not a good practice to add `using namespace` in header file.
Instead use `std::` directly where needed